NebulaWorld/Warning/WarningManager.cs

```diff
@@ -175,9 +175,14 @@ public void ImportBinaryData(BinaryReader br, int activeWarningCount)
 
             // reassign warningId for trash
             var trashId = br.ReadInt32();
-            if (trashId >= 0 && trashId < GameMain.data.trashSystem.container.trashCursor)
+            var continaer = GameMain.data.trashSystem.container;
+            if (trashId >= 0 && trashId < continaer.trashCursor)
             {
-                GameMain.data.trashSystem.container.trashDataPool[trashId].warningId = i;
+                // assign warningId only if item exist
+                if (continaer.trashObjPool[trashId].item > 0)
+                {
+                    continaer.trashDataPool[trashId].warningId = i;
+                }
             }
```

Comment:
Minor nitpick, these 2 ifs could be collapsed if you want?

Reply:
or it could be inverted (and fix typo on variable name continaer -> container 
```cs
var container = GameMain.data.trashSystem.container;
if (trashId < 0 || trashId >= container .trashCursor)
{
    continue;
}
// assign warningId only if item exist
if (container .trashObjPool[trashId].item > 0)
{
    container .trashDataPool[trashId].warningId = i;
}
```